From 68f993e214e2e35108cf492dc259ca45940829bf Mon Sep 17 00:00:00 2001
From: Rob Herring <r.herring@freescale.com>
Date: Mon, 6 Apr 2009 05:55:37 -0500
Subject: [PATCH 091/449] ENGR00110997: Change mx51 platform name to imx51

Change mx51 platform name to imx51 so tests will build for all
mx51 platforms.
Remove old MXC platform stuff.

Signed-off-by: Rob Herring <r.herring@freescale.com>
---
 test-utils.sh                    |   5 +-
 test/max8660_pmic_test/Makefile  |   2 +-
 test/mxc_asrc_test/Makefile      |   2 +-
 test/mxc_dryice_test/Makefile    |   2 +-
 test/mxc_firi_test/Makefile      |   2 +-
 test/mxc_hdd_test/Makefile       |   2 +-
 test/mxc_i2c_slave_test/Makefile |   2 +-
 test/mxc_ipudev_test/Makefile    |   2 +-
 test/mxc_mlb_test/Makefile       |   2 +-
 test/mxc_mpeg4_enc_test/Makefile |   2 +-
 test/mxc_mu_test/Makefile        |  22 ----
 test/mxc_mu_test/autorun-mu.sh   |  21 ----
 test/mxc_mu_test/mxc_mu_test.c   | 252 ---------------------------------------
 test/mxc_oprofile_test/Makefile  |   2 +-
 test/mxc_owire_test/Makefile     |   2 +-
 test/mxc_pf_test/Makefile        |   2 +-
 test/mxc_pmic_test/Makefile      |   2 +-
 test/mxc_rng_test/Makefile       |  11 +-
 test/mxc_rtc/autorun-rtc.sh      |   2 +-
 test/mxc_sahara_test/Makefile    |   2 +-
 test/mxc_vpu_test/Makefile       |   2 +-
 21 files changed, 19 insertions(+), 324 deletions(-)
 delete mode 100644 test/mxc_mu_test/Makefile
 delete mode 100755 test/mxc_mu_test/autorun-mu.sh
 delete mode 100644 test/mxc_mu_test/mxc_mu_test.c

diff --git a/test-utils.sh b/test-utils.sh
index 41cd13f..a9012cd 100755
--- a/test-utils.sh
+++ b/test-utils.sh
@@ -146,9 +146,8 @@ print_status()
 
 #
 # This echos the plaftorm name in all caps:
-# MXC27530EVB, MXC27520EVB, IMX27ADS, IMX31ADS, MXC91131EVB, MXC30030EVB, MXC30020EVB
 #
-# for example use in an if statement like:  if [ "$(platform)" = MXC27530EVB ]; then
+# for example use in an if statement like:  if [ "$(platform)" = IMX25_3STACK ]; then
 #
 # This is only for use where a test needs to run differently for different platforms.
 # To keep a test from running at all for a platform, add it to the EXCLUDES list in
@@ -172,7 +171,7 @@ platform()
 		MX313StackBoard )	plat=IMX31_3STACK ;;
 		MX353StackBoard )	plat=IMX35_3STACK ;;
 		MX373StackBoard )	plat=IMX37_3STACK ;;
-		MX513StackBoard )       plat=IMX51_3STACK ;;
+		MX51* )       		plat=IMX51 ;;
 
 		* )     ;;
 	esac
diff --git a/test/max8660_pmic_test/Makefile b/test/max8660_pmic_test/Makefile
index 81c73dd..f66a01f 100644
--- a/test/max8660_pmic_test/Makefile
+++ b/test/max8660_pmic_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX51_3STACK IMX37_3STACK IMX31_3STACK IMX31ADS IMX27_3STACK IMX27ADS IMX25_3STACK"
+EXCLUDE_LIST:="IMX51 IMX37_3STACK IMX31_3STACK IMX31ADS IMX27_3STACK IMX27ADS IMX25_3STACK"
 
 
 LTP_DIR = $(TOPDIR)/test/mxc_pmic_test/
diff --git a/test/mxc_asrc_test/Makefile b/test/mxc_asrc_test/Makefile
index 0f8cd99..08d9ad7 100644
--- a/test/mxc_asrc_test/Makefile
+++ b/test/mxc_asrc_test/Makefile
@@ -2,7 +2,7 @@
 # Makefile for OSS sound driver test app.
 #
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX51_3STACK IMX25_3STACK
+EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX51 IMX25_3STACK
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = \
diff --git a/test/mxc_dryice_test/Makefile b/test/mxc_dryice_test/Makefile
index 2dc9868..5f68a25 100644
--- a/test/mxc_dryice_test/Makefile
+++ b/test/mxc_dryice_test/Makefile
@@ -5,7 +5,7 @@
 #****************************************************************************/
 
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX27EVB IMX31ADS IMX31_3STACK IMX35_3STACK IMX37_3STACK IMX51_3STACK"
+EXCLUDE_LIST:="IMX27ADS IMX27EVB IMX31ADS IMX31_3STACK IMX35_3STACK IMX37_3STACK IMX51"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/dryice.sh
diff --git a/test/mxc_firi_test/Makefile b/test/mxc_firi_test/Makefile
index 072bd8a..6b4f171 100644
--- a/test/mxc_firi_test/Makefile
+++ b/test/mxc_firi_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="MXC27520EVB MXC27530EVB IMX27ADS MXC30020EVB MXC30031ADS IMX37_3STACK IMX35_3STACK IMX51_3STACK IMX25_3STACK"
+EXCLUDE_LIST:="IMX27ADS IMX37_3STACK IMX35_3STACK IMX51 IMX25_3STACK"
 
 # list of platforms which do not have either SIR or FIR
 IR_EXCLUDE_LIST:=
diff --git a/test/mxc_hdd_test/Makefile b/test/mxc_hdd_test/Makefile
index 282859e..da964b7 100644
--- a/test/mxc_hdd_test/Makefile
+++ b/test/mxc_hdd_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="MXC91131EVB I30030ADS I30030EVB IMX21ADS MXC27520EVB MXC27530EVB MXC30020EVB MXC30030EVB MXC30031ADS IMX25_3STACK"
+EXCLUDE_LIST:="IMX21ADS IMX25_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-ata.sh
diff --git a/test/mxc_i2c_slave_test/Makefile b/test/mxc_i2c_slave_test/Makefile
index 0cdc2fc..13d87d6 100644
--- a/test/mxc_i2c_slave_test/Makefile
+++ b/test/mxc_i2c_slave_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:=IMX31ADS IMX27ADS IMX37_3STACK IMX31_3STACK IMX51_3STACK
+EXCLUDE_LIST:=IMX31ADS IMX27ADS IMX37_3STACK IMX31_3STACK IMX51
 
 CFLAGS += -lpthread
 
diff --git a/test/mxc_ipudev_test/Makefile b/test/mxc_ipudev_test/Makefile
index 4a2c093..136e525 100644
--- a/test/mxc_ipudev_test/Makefile
+++ b/test/mxc_ipudev_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which want this test case
-INCLUDE_LIST:=IMX31_3STACK IMX35_3STACK IMX37_3STACK IMX51_3STACK
+INCLUDE_LIST:=IMX31_3STACK IMX35_3STACK IMX37_3STACK IMX51
 
 SRCS = mxc_ipudev_test.c utils.c test_pattern.c
 CFLAGS += -lipu -lIpuScreenLayer -lm
diff --git a/test/mxc_mlb_test/Makefile b/test/mxc_mlb_test/Makefile
index 68c5c82..51a4253 100644
--- a/test/mxc_mlb_test/Makefile
+++ b/test/mxc_mlb_test/Makefile
@@ -2,7 +2,7 @@
 # Makefile for MediaLB driver test app.
 #
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX51_3STACK IMX25_3STACK
+EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX51 IMX25_3STACK
 
 INC += -I$(LINUXPATH)/include
 
diff --git a/test/mxc_mpeg4_enc_test/Makefile b/test/mxc_mpeg4_enc_test/Makefile
index a49933e..beba194 100644
--- a/test/mxc_mpeg4_enc_test/Makefile
+++ b/test/mxc_mpeg4_enc_test/Makefile
@@ -5,7 +5,7 @@ OBJDIR=$(TOPDIR)/platform/$(PLATFORM)/$(MODULE_NAME)
 CFLAGS += -I . -lpthread
 
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX37_3STACK IMX35_3STACK IMX51_3STACK IMX25_3STACK"
+EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX37_3STACK IMX35_3STACK IMX51 IMX25_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 EXES = $(OBJDIR)/cam2mpeg4.out
diff --git a/test/mxc_mu_test/Makefile b/test/mxc_mu_test/Makefile
deleted file mode 100644
index ed7783b..0000000
--- a/test/mxc_mu_test/Makefile
+++ /dev/null
@@ -1,22 +0,0 @@
-# list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX31ADS IMX37_3STACK IMX35_3STACK IMX51_3STACK IMX25_3STACK"
-
-ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
-OBJS = \
-	$(OBJDIR)/mxc_mu_test.out \
-        $(OBJDIR)/autorun-mu.sh
-else
-OBJS =
-endif
-
-all : $(OBJS)
-
-.PHONY: clean
-clean :
-	rm -f $(OBJS)
-
-#
-# include the Rules
-#
-include ../make.rules
-
diff --git a/test/mxc_mu_test/autorun-mu.sh b/test/mxc_mu_test/autorun-mu.sh
deleted file mode 100755
index c5e6f3d..0000000
--- a/test/mxc_mu_test/autorun-mu.sh
+++ /dev/null
@@ -1,21 +0,0 @@
-#!/bin/sh
-
-source /unit_tests/test-utils.sh
-
-#
-# Exit status is 0 for PASS, nonzero for FAIL
-#
-STATUS=0
-platform=`cat /proc/cpuinfo | grep Hardware |cut -f3 -d' '`
- if  [ $platform = "MXC300-31" ]  || [ $platform = "MXC300-20"  ]
- then
- m2ramaddr=10000100
- else
- m2ramaddr=ff002800
-fi
-#echo M2RAM Address is: $m2ramaddr
-# MU protocol test case
-echo $m2ramaddr | run_testcase "./mxc_mu_test.out "
-
-print_status
-exit $STATUS
diff --git a/test/mxc_mu_test/mxc_mu_test.c b/test/mxc_mu_test/mxc_mu_test.c
deleted file mode 100644
index fa80790..0000000
--- a/test/mxc_mu_test/mxc_mu_test.c
+++ /dev/null
@@ -1,252 +0,0 @@
-/*
- * Copyright 2006 Freescale Semiconductor, Inc. All rights reserved.
- */
-
-/*
- * The code contained herein is licensed under the GNU General Public
- * License. You may obtain a copy of the GNU General Public License
- * Version 2 or later at the following locations:
- *
- * http://www.opensource.org/licenses/gpl-license.html
- * http://www.gnu.org/copyleft/gpl.html
- */
-
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include <sys/ioctl.h>
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <sys/time.h>
-#include <sys/wait.h>
-#include <fcntl.h>
-#include <unistd.h>
-#include <poll.h>
-#include "../include/mxc_test.h"
-
-int main(int argc, char **argv)
-{
-	int fp[4], bytes_written = 0, i, j, bytes_read = 0;
-	int flush;
-	unsigned long M2RAM_Phy_Addr;
-	unsigned int platform;
-	struct pollfd fds;
-	unsigned int ret;
-	int write_status = 0, read_status = 0, poll_read = 0, poll_write = 0;
-
-	unsigned int writereq = 0x00050001;
-	unsigned int readreq = 0x00050002;
-	unsigned int address = 0xff002800;
-	printf("================== TESTING MU DRIVER ==================\n");
-	printf
-	    ("Please enter M2RAM Physical address for this platform e.g ff000000 \n");
-	scanf("%x", &M2RAM_Phy_Addr);
-	address = M2RAM_Phy_Addr;
-	/*
-	   printf("Please specify Hardware platform..... \n");
-	   printf(" 1 = mxc27520evb \n");
-	   printf(" 2 = mxc27530evb \n");
-	   printf(" 3 = mxc30030\n");
-	   printf(" 4 = mxc30020evb \n");
-	   printf(" 5 = mxc30030evb \n");
-	   printf(" 6 = mxc91131evb \n" );
-	   scanf("%d", &platform);
-	   printf("Platform value in application is: %x \n", platform);
-	 */
-	unsigned int datavalue1 = 0xaaaaaaa1;
-	unsigned int datavalue2 = 0xbbbbbbb2;
-	unsigned int datavalue3 = 0xccccccc3;
-	unsigned int datavalue4 = 0xddddddd4;
-	unsigned int datavalue5 = 0xeeeeeee5;
-	unsigned char writecode[4], addr[4], data1[4], data2[4];
-	unsigned char data3[4], data4[4], data5[4], readcode[4];
-	unsigned char user_buff[1000];
-	memset(user_buff, '\0', 1000);
-	for (j = 3; j >= 0; j--) {
-		writecode[j] = writereq >> (j * 8);
-	}
-	for (j = 3; j >= 0; j--) {
-		readcode[j] = readreq >> (j * 8);
-	}
-	for (j = 3; j >= 0; j--) {
-		addr[j] = address >> (j * 8);
-	}
-	for (j = 3; j >= 0; j--) {
-		data1[j] = datavalue1 >> (j * 8);
-	}
-	for (j = 3; j >= 0; j--) {
-		data2[j] = datavalue2 >> (j * 8);
-	}
-	for (j = 3; j >= 0; j--) {
-		data3[j] = datavalue3 >> (j * 8);
-	}
-	for (j = 3; j >= 0; j--) {
-		data4[j] = datavalue4 >> (j * 8);
-	}
-	for (j = 3; j >= 0; j--) {
-		data5[j] = datavalue5 >> (j * 8);
-	}
-
-	printf("===================== OPEN Device =====================\n");
-	fp[0] = open("/dev/mxc_mu/0", O_RDWR | O_NONBLOCK);
-	fp[1] = open("/dev/mxc_mu/1", O_RDWR | O_NONBLOCK);
-	fp[2] = open("/dev/mxc_mu/2", O_RDWR | O_NONBLOCK);
-	fp[3] = open("/dev/mxc_mu/3", O_RDWR | O_NONBLOCK);
-	/* Enabling Byte swapping on read */
-	ioctl(fp[0], 0x8);
-	ioctl(fp[1], 0x8);
-	ioctl(fp[2], 0x8);
-	ioctl(fp[3], 0x8);
-
-	if (fp[0] < 0 || fp[1] < 0 || fp[2] < 0 || fp[3] < 0) {
-		printf("Open failed with err %d\n", fp[0]);
-		perror("Open");
-		return -1;
-	} else {
-		printf("Open Device is %d\n", fp[0]);
-		/*Need to determine why dummy read is required */
-		/*the test works without it */
-		//if((platform==mxc91231) || (platform==mxc91311)){
-		//printf("Doing a dummy read\n");
-		//read(fp[0], user_buff, 4);
-		//memset(user_buff, '\0', 1000);
-		//      }
-		printf
-		    ("================== After OPEN Device =================\n");
-		printf
-		    ("=============== Before Calling Write =================\n");
-		printf("Now calling 1st write\n");
-		bytes_written = write(fp[0], writecode, 4);	//0x12345678 = 4 bytes                                                                                   //each digit pos = 4 binary bits
-		printf("Total Bytes actually written are %d\n", bytes_written);
-		printf("Now calling 2nd write\n");
-		bytes_written = write(fp[1], addr, 4);
-		printf("Total Bytes actually written are %d\n", bytes_written);
-		printf("Now calling data writes\n");
-		bytes_written = write(fp[2], data1, 4);
-		bytes_written = write(fp[3], data2, 4);
-		bytes_written = write(fp[0], data3, 4);
-		bytes_written = write(fp[1], data4, 4);
-		bytes_written = write(fp[2], data5, 4);
-		/* Read write response */
-		bytes_read = read(fp[0], user_buff, 4);
-		for (i = 0; i < bytes_read; i++) {
-			printf("Read Buffer after write response is %x\n",
-			       user_buff[i]);
-		}
-		if ((user_buff[0] == 0x00) && (user_buff[1] == 0x05)
-		    && (user_buff[2] == 0x00) && (user_buff[3] == 0x01)) {
-			write_status = 1;
-		}
-		//printf("Press key when ready... \n");
-		//scanf("%d",&i);
-		/* Sending Read Request */
-		bytes_written = write(fp[0], readcode, 4);
-		bytes_written = write(fp[1], addr, 4);
-		printf
-		    ("================ Before Calling all Reads  =================\n");
-		bytes_read = read(fp[1], user_buff, 4);
-		for (i = 0; i < bytes_read; i++) {
-			printf("Read Buffer after read 1 from MRR1 is %x\n",
-			       user_buff[i]);
-		}
-		bytes_read = read(fp[2], user_buff, 4);
-		for (i = 0; i < bytes_read; i++) {
-			printf("Read Buffer after read 2 from MRR2 is %x\n",
-			       user_buff[i]);
-		}
-		bytes_read = read(fp[3], user_buff, 4);
-		for (i = 0; i < bytes_read; i++) {
-			printf("Read Buffer after read 3 from MRR3 is %x\n",
-			       user_buff[i]);
-		}
-		bytes_read = read(fp[0], user_buff, 4);
-		bytes_read = read(fp[0], user_buff, 4);
-		for (i = 0; i < bytes_read; i++) {
-			printf("Read Buffer after read 4 from MRR0 is %x\n",
-			       user_buff[i]);
-		}
-		bytes_read = read(fp[1], user_buff, 4);
-		for (i = 0; i < bytes_read; i++) {
-			printf("Read Buffer after read 5 from MRR1 is %x\n",
-			       user_buff[i]);
-		}
-		/* Read read response */
-		bytes_read = read(fp[0], user_buff, 4);
-		for (i = 0; i < bytes_read; i++) {
-			printf("Read Buffer after read response is %x\n",
-			       user_buff[i]);
-		}
-		if ((user_buff[0] == 0x00) && (user_buff[1] == 0x05)
-		    && (user_buff[2] == 0x00) && (user_buff[3] == 0x02)) {
-			read_status = 1;
-		}
-		printf
-		    ("\n ####### TESTING POLLING FEATURE on MU channel 3 #######\n");
-		printf
-		    ("\nNow that all the data are transmitted and received, \n");
-		printf
-		    ("a POLL on a particular channel should indicate no data\n");
-		printf
-		    ("to be read and more data can be transmitted i.e., buffer empty \n\n");
-		fds.fd = fp[3];
-		fds.events = POLLIN;
-		ret = poll(&fds, 1, 0);
-		if (ret == 0) {
-			printf
-			    ("READ: POLLIN - Timeout, No data to read using channel 3.\n\n");
-			poll_read = 1;
-		} else {
-			bytes_read = read(fp[3], user_buff, 4);
-			for (i = 0; i < bytes_read; i++) {
-				printf
-				    ("Read Buffer after read 3 from MRR3 is %x\n\n",
-				     user_buff[i]);
-			}
-		}
-		/* Now change the events to WRITE */
-		fds.events = POLLOUT;
-		ret = poll(&fds, 1, 0);
-		if (ret == 0) {
-			printf
-			    ("WRITE: POLLOUT - Timeout, No space to write.using channel 3\n\n");
-		} else {
-			printf
-			    ("WRITE: POLLOUT - Space Available in write buffer.\n");
-			printf
-			    ("So data can be transmitted using channel 3\n\n");
-			poll_write = 1;
-		}
-		printf("Now closing all the channels\n\n");
-		flush = close(fp[0]);
-		if (flush < 0) {
-			printf("Close failed with err %d\n", flush);
-			perror("Close");
-			return -1;
-		}
-		flush = close(fp[1]);
-		if (flush < 0) {
-			printf("Close failed with err %d\n", flush);
-			perror("Close");
-			return -1;
-		}
-		flush = close(fp[2]);
-		if (flush < 0) {
-			printf("Close failed with err %d\n", flush);
-			perror("Close");
-			return -1;
-		}
-		flush = close(fp[3]);
-		if (flush < 0) {
-			printf("Close failed with err %d\n", flush);
-			perror("Close");
-			return -1;
-		}
-	}
-	if ((write_status > 0) && (read_status > 0) && (poll_write > 0)
-	    && (poll_read > 0)) {
-		printf("MU Test PASSED\n\n");
-		return 0;
-	} else {
-		return -1;
-	}
-}
diff --git a/test/mxc_oprofile_test/Makefile b/test/mxc_oprofile_test/Makefile
index aadc9ba..d21a702 100644
--- a/test/mxc_oprofile_test/Makefile
+++ b/test/mxc_oprofile_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="MXC27520EVB IMX27ADS MXC30020EVB"
+EXCLUDE_LIST:=""
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-oprofile.sh
diff --git a/test/mxc_owire_test/Makefile b/test/mxc_owire_test/Makefile
index d8ceda6..e218b00 100644
--- a/test/mxc_owire_test/Makefile
+++ b/test/mxc_owire_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX31_3STACK IMX37_3STACK IMX35_3STACK IMX51_3STACK IMX25_3STACK"
+EXCLUDE_LIST:="IMX31_3STACK IMX37_3STACK IMX35_3STACK IMX51 IMX25_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-owire.sh
diff --git a/test/mxc_pf_test/Makefile b/test/mxc_pf_test/Makefile
index 9cf473c..625f9d7 100644
--- a/test/mxc_pf_test/Makefile
+++ b/test/mxc_pf_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX37_3STACK IMX51_3STACK IMX25_3STACK"
+EXCLUDE_LIST:="IMX27ADS IMX37_3STACK IMX51 IMX25_3STACK"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/autorun-pf.sh $(OBJDIR)/mxc_pf_test.out
diff --git a/test/mxc_pmic_test/Makefile b/test/mxc_pmic_test/Makefile
index 3fb8dd8..062cc9c 100644
--- a/test/mxc_pmic_test/Makefile
+++ b/test/mxc_pmic_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX37_3STACK IMX35_3STACK IMX51_3STACK"
+EXCLUDE_LIST:="IMX37_3STACK IMX35_3STACK IMX51"
 
 LTP_DIR = ../
 LOADLIBES+= -L$(LTP_DIR)lib -lltp -lpthread -lm
diff --git a/test/mxc_rng_test/Makefile b/test/mxc_rng_test/Makefile
index fe7cc49..3d8b73e 100644
--- a/test/mxc_rng_test/Makefile
+++ b/test/mxc_rng_test/Makefile
@@ -5,7 +5,7 @@
 #****************************************************************************/
 
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX27ADS IMX27EVB MXC27530EVB MXC30030ADS MXC30030EVB MXC30031ADS IMX51_3STACK"
+EXCLUDE_LIST:="IMX27ADS IMX51"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = apitest rng_test move
@@ -13,15 +13,6 @@ else
 OBJS =
 endif
 KERNEL_DIR = $(LINUXPATH)
-ifeq ("$(PLATFORM)", "MXC27520EVB")
-TARGET_ARCH=-DCONFIG_ARCH_MXC91221
-endif
-ifeq ("$(PLATFORM)", "MXC30020EVB")
-TARGET_ARCH=-DCONFIG_ARCH_MXC91311
-endif
-ifeq ("$(PLATFORM)", "MXC91131EVB")
-TARGET_ARCH=-DCONFIG_ARCH_MXC91131
-endif
 ifeq ("$(PLATFORM)", "IMX25_3STACK")
 TARGET_ARCH=-DCONFIG_ARCH_MX25
 endif
diff --git a/test/mxc_rtc/autorun-rtc.sh b/test/mxc_rtc/autorun-rtc.sh
index 42293b5..4e934ab 100755
--- a/test/mxc_rtc/autorun-rtc.sh
+++ b/test/mxc_rtc/autorun-rtc.sh
@@ -8,7 +8,7 @@ source /unit_tests/test-utils.sh
 STATUS=0
 
 echo "$(platform)"
-if [ "$(platform)" == IMX37_3STACK ] || [ "$(platform)" == IMX51_3STACK ]; then
+if [ "$(platform)" == IMX37_3STACK ] || [ "$(platform)" == IMX51 ]; then
 	rtc_test_param=--no-periodic
 	RTC_IRQS_EXPECTED=1
 else
diff --git a/test/mxc_sahara_test/Makefile b/test/mxc_sahara_test/Makefile
index 748cb38..5a92736 100644
--- a/test/mxc_sahara_test/Makefile
+++ b/test/mxc_sahara_test/Makefile
@@ -7,7 +7,7 @@ ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 ifeq ("$(PLATFORM)", "IMX27ADS")
 TARGET_ARCH=-DCONFIG_ARCH_MX27
 endif
-ifeq ("$(PLATFORM)", "IMX51_3STACK")
+ifeq ("$(PLATFORM)", "IMX51")
 TARGET_ARCH=-DCONFIG_ARCH_MX51
 endif
 
diff --git a/test/mxc_vpu_test/Makefile b/test/mxc_vpu_test/Makefile
index 28d024d..50cbce6 100644
--- a/test/mxc_vpu_test/Makefile
+++ b/test/mxc_vpu_test/Makefile
@@ -11,7 +11,7 @@ OBJ = main.o \
       loopback.o
 
 # list of platforms which want this test case
-INCLUDE_LIST:=IMX27ADS IMX31ADS IMX37_3STACK IMX51_3STACK IMX51
+INCLUDE_LIST:=IMX27ADS IMX31ADS IMX37_3STACK IMX51
 
 ifeq ($(PLATFORM),$(findstring $(PLATFORM),$(INCLUDE_LIST)))
 TARGET = mxc_vpu_test.out
-- 
1.8.0

